src/ClientData/include/ClientData/CallstackData.h

```diff
@@ -75,6 +76,19 @@ class CallstackData {
     }
   }
 
+  // Do a particular action for callstacks but skipping callstacks that will be rendered later in
+  // the same pixel in the screen. It assures to do the action once for callstacks in each occupied
+  // pixel. This iteration is faster that the non-discretized one since it doesn't require going
```

Comment:
I think "once for callstacks in each occupied pixel" is still confusing. Once per callstack or once per pixel?

Reply:
Same below.